docs/product-user-guide/src/main/asciidoc/decision-tables-defining-proc.adoc

```diff
@@ -0,0 +1,47 @@
+[id='decision-tables-defining-proc']
+= Defining decision tables
+
+Decision table spreadsheets (XLS or XLSX) require two key areas that define rule data: a `RuleSet` area and a `RuleTable` area. The `RuleSet` area of the spreadsheet defines elements that you want to apply globally to all rules in the same package (not only the spreadsheet), such as a rule set name or universal rule attributes. The `RuleTable` area defines the actual rules (rows) and the conditions, actions, and other rule attributes (columns) that constitute that rule table within the specified rule set. A decision table spreadsheet can contain multiple `RuleTable` areas, but only one `RuleSet` area.
+
+IMPORTANT: You should typically create only one decision table spreadsheet, containing all necessary `RuleTable` definitions, per rule package in {CENTRAL}. You can create separate decision table spreadsheets for separate packages, but creating multiple decision table spreadsheets in the same package can cause compilation errors from conflicting `RuleSet` or `RuleTable` attributes and is therefore not recommended.
+
+Refer to the following sample spreadsheet as you define your decision table:
+
+.Sample decision table spreadsheet
+image::1248.jpg[]
+
+.Procedure
+. In a new XLS or XLSX spreadsheet, go to the second or third column and label a cell `RuleSet` (row 9 in example). Reserve the column or columns to the left for descriptive metadata (optional).
+. In the next cell to the right, enter a name for the `RuleSet`. This named rule set will contain all `RuleTable` rules defined in the rule package.
+. Under the `RuleSet` cell, define any rule attributes (one per cell) that you want to apply globally to all rule tables in the package. Specify attribute values in the cells to the right. For supported cell labels and values, see xref:decision-tables-rule-set-entries-ref[].
+. Below the the `RuleSet` area and in the same column as the `RuleSet` cell, skip a row and label a new cell `RuleTable` (row 13 in example) and enter a table name in the same cell. The name string is used as the initial part of the name for all rules derived from this rule table, with the row number appended for distinction. You can override this automatic naming by inserting a `NAME` attribute column.
```

Comment:
"The name string is used".. Probably fine, but may be better to say "The name (type: string) is used"..